Mathlib/GroupTheory/PresentedGroup.lean

```diff
@@ -27,13 +27,14 @@ given by generators `x : α` and relations `r ∈ rels`.
 generators, relations, group presentations
 -/
 
+open FreeGroup Subgroup
 
 variable {α : Type*}
 
 /-- Given a set of relations, `rels`, over a type `α`, `PresentedGroup` constructs the group with
 generators `x : α` and relations `rels` as a quotient of `FreeGroup α`. -/
 def PresentedGroup (rels : Set (FreeGroup α)) :=
-  FreeGroup α ⧸ Subgroup.normalClosure rels
+  FreeGroup α ⧸ normalClosure rels
```

Comment:
For a `feat` PR, it's better practice to not mix refactoring into it, unless you can find some reviewer who thinks it's immediately a good idea and is OK with the distraction of interleaved non-feature changes.

I'm not sure removing `Subgroup.` and `FreeGroup.` is an improvement myself, but I don't know that it's not an improvement either. If you limit the PR to just adding `equivPresentedGroup` you'll have a better shot at getting it quickly reviewed.

Reply:
@kmill Understood.  I will revert the last commit.